src/SFA.DAS.PSRService.Web/Services/PeriodService.cs

```diff
@@ -1,36 +1,30 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Globalization;
-using MediatR;
-using SFA.DAS.PSRService.Application.ReportHandlers;
 using SFA.DAS.PSRService.Domain.Entities;
-using SFA.DAS.PSRService.Domain.Enums;
-using SFA.DAS.PSRService.Web.Configuration;
-using SFA.DAS.PSRService.Web.ViewModels;
-using SFA.DAS.PSRService.Web.Models;
 
 namespace SFA.DAS.PSRService.Web.Services
 {
     public class PeriodService : IPeriodService
     {
+        private readonly IDateTimeService _dateTimeService;
 
-        private IWebConfiguration _config;
-
-        public PeriodService(IWebConfiguration config)
+        public PeriodService(IDateTimeService dateTimeService)
         {
-            _config = config;
+            _dateTimeService = dateTimeService ?? throw new ArgumentNullException(nameof(dateTimeService));
         }
 
-
         public Period GetCurrentPeriod()
         {
-            return new Period(DateTime.UtcNow);
+            return Period.FromInstantInPeriod(_dateTimeService.UtcNow);
         }
 
-        public bool IsSubmissionsOpen()
+        public bool ReportIsForCurrentPeriod(Report report)
```

Comment:
The PeriodService should not take a report - this method could be changed to take a period instead. The call from ReportService can then pass report.Period to it.